### PR TITLE
feat/sharing functionality

### PR DESCRIPTION
**Pull request by alper92**
#11 

Reference: https://www.npmjs.com/package/react-share

### CODE REVIEW

File: components/ArtPieceDetails/ArtPieceDetails.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
+import { faShareNodes } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import Image from "next/image";
 import Link from "next/link";
+import { FacebookIcon, WhatsappIcon, WhatsappShareButton } from "react-share";
+import { FacebookShareButton } from "react-share";
```
Comment (Roland-Hufnagel) [inline components/ArtPieceDetails/ArtPieceDetails.js:6]: Please. put them together.

File: components/ArtPieceDetails/ArtPieceDetails.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
+import { faShareNodes } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
```
Comment (Roland-Hufnagel) [inline components/ArtPieceDetails/ArtPieceDetails.js:2]: where do you use them?

File: components/ArtPieceDetails/ArtPieceDetails.js [hunk 3]
```diff
@@ -27,11 +32,37 @@ export default function ArtPieceDetails({
...
+          </WhatsappShareButton>
+        </WhatsappItem>
+      </ButtonList>
       <StyledCaption>
         <StyledNameDate>
-          <p>{name}</p>
-          {date}
+          <b>
+            <q>{name}</q>
+          </b>
```
Comment (Roland-Hufnagel) [inline components/ArtPieceDetails/ArtPieceDetails.js:64]: ``<b>`` and ``<q>``are not good choices (read here: 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b?retiredLocale=de and here:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/q?retiredLocale=de)
A headline could be a good alternative.

File: components/CategoryFilter/CategoryFilter.js [hunk 4]
```diff
@@ -19,38 +19,46 @@ export default function CategoryFilter({ artPieces, handleSetFilteredCategory })
   }
 
   return (
-    <StyledCategoryFilter>
-      <StyledNewestButton onClick={handleNewestArtPieces}>Newest</StyledNewestButton>
-      {uniqueCatagories.map((category) => (
-        <li key={category}>
-          <StyledButton onClick={() => handleFilteredCategories(category)}>{category}</StyledButton>
+    <StyledNavigaton>
```
Comment (Roland-Hufnagel) [inline components/CategoryFilter/CategoryFilter.js:22]: Since you don't have links you should not use a ``<nav>``tag.

**Review by Roland-Hufnagel — CHANGES_REQUESTED** [review]
Hi Alper. Your PR looks really great 💪 . Even if I don't know if it will pass the QA. I don't know the react-share library and sharing with whatsapp didn't work the way I expected. Nevertheless I left some comments I want you to look at...

**Comment by alper92** [comment]
> Hi Alper. Your PR looks really great 💪 . Even if I don't know if it will pass the QA. I don't know the react-share library and sharing with whatsapp didn't work the way I expected. Nevertheless I left some comments I want you to look at...

Thank you very much for your review! I fixed the things you pointed out. I guess the sharing did not work as expected sind you are not logged in in Facebook or Whatsapp. I tried it out on my phone and the mac browser.

File: components/ArtPiecesPreview/ArtPiecesPreview.js [hunk 5]
```diff
@@ -23,7 +23,9 @@ export default function ArtPiecesPreview({ artPieces, filteredCategory }) {
                 />
               </Link>
               <Caption>
-                <p>{name}</p>
+                <b>
+                  <q>{name}</q>
+                </b>
```
Comment (Roland-Hufnagel) [inline components/ArtPiecesPreview/ArtPiecesPreview.js:28]: Here is this thing again.

File: components/CategoryFilter/CategoryFilter.js (hunk 4 above)
Comment (Roland-Hufnagel) [reply components/CategoryFilter/CategoryFilter.js:26]: For good naming convention please now call it StyledSection.

File: components/ArtPieceDetails/ArtPieceDetails.js [hunk 6]
```diff
@@ -27,11 +29,35 @@ export default function ArtPieceDetails({
...
+          >
+            <WhatsappIcon size={32} round={true} />
+          </WhatsappShareButton>
+        </li>
+      </ButtonList>
       <StyledCaption>
         <StyledNameDate>
-          <p>{name}</p>
-          {date}
+          <h2>{name}</h2>
```
Comment (Roland-Hufnagel) [inline components/ArtPieceDetails/ArtPieceDetails.js:59]: There has to be a  ``<h1>`` on the page before you can use a ``<h2>``. Either make sure, you have a title or something as a h1 on your page or use the h1 here or use a p-tag and style it like you want.

**Review by Roland-Hufnagel — APPROVED** [review]
Hi. I can approve now but there are still some smaller points. Please look at my comments once more. 
For the QA: I am using WhatsApp on my macbook but your site asks me to open WhatsApp in my phone and scan the QR-Code.  But thats a point for the QA.

**Pull request merged**